samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir/docs/FAQ.md

```diff
@@ -0,0 +1,104 @@
+# FAQ's for FHIR-to-FHIR data movement
+
+1. How do we perform FHIR to FHIR (Gen1 to Gen1) data transfer?
+   + For implementation details and architecture overview please refer the [documentation](https://github.com/Azure-Samples/azure-health-data-services-samples/tree/main/samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir#copying-data-from-one-azure-api-for-fhir-server-to-another-azure-api-for-fhir-server).
+
+2. How to deploy FHIR loader?
+   + Go to [FHIR Bulk Loader](https://github.com/microsoft/fhir-loader) and follow the [deployment steps](https://github.com/microsoft/fhir-loader#deployment).
+
+3. How to configure Azure API for FHIR server for export?
+   + Follow steps [here](https://learn.microsoft.com/en-us/azure/healthcare-apis/azure-api-for-fhir/configure-export-data) to configure settings for export on Azure API for FHIR server.
+
+4. Will there be any message to the user when export is finished?
+   + No, users must check the status manually using the get request in Postman with URL returned in the Content-Location response headers of export operation response.
+     
+     If the URL returns status 202 Accepted, it means the export job is still in progress, if the status is 200 Completed, it means the export job is completed.
+     
+     More details could be found in the readme document of the app [here](https://github.com/Azure-Samples/azure-health-data-services-samples/tree/main/samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir).
+
+5. How do you check when the FHIR loader is done uploading?
+   + Users must check the containers i.e., "bundles" and "ndjson� to know about FHIR loader completion. FHIR Loader has completed loading when these containers are empty.
```

Comment:
The character after ndjson seems not a standard quote.

Reply:
changes done as per review comments.